shared/redux/rtk-apis/assignment_submissions/assignment-submissions.api.ts

```diff
@@ -5,6 +5,14 @@ import { TSubmission } from "./assignment-submissions.types";
 
 const assignmentSubmissionsApi = projectApi.injectEndpoints({
   endpoints: (builder) => ({
+    getSubmissions: builder.query<TSubmission[], number>({
+      query: (assignmentId) => `assignments/${assignmentId}/submissions`,
```

Comment:
Let's follow REST principles


```suggestion
      query: (assignmentId) => `classrooms/:classroomId/assignments/${assignmentId}/submissions`,
```